.github/workflows/qa-checks.yml

```diff
@@ -21,9 +21,6 @@ jobs:
         run: npm install
 
       - name: Linters and Formatters Check
-        uses: wearerequired/lint-action@v1
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          # Enable linters
-          eslint: true
-          prettier: true
+        run: |
+         npm run lint
+         npm run check
```

Comment:
@Amulya-coder Please add the script for prettier check in package.json.
Checks are also failing due to this.